outputs.tf

```diff
@@ -1,8 +1,8 @@
-##############################################################################
+########################################################################################################################
 # Outputs
-##############################################################################
+########################################################################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "tenant_id" {
+  description = "AppID instance guid, also called as tenant_id."
+  value       = ibm_resource_instance.appid.guid
+}
```

Comment:
can you expose more outputs here? name, id, guid, crn etc etc

Reply:
I don't see any new outputs added?

Reply:
I have already added `id` & `dashboard_url` output variables. `tenant_id` is GUID.